Microsoft/WebTemplateStudio
# Autofill name field in Azure forms

closes #983 

**How It Works**
- user opens Azure Services form
- when a subscription is selected, we send the project name to the extension
- extension returns a valid name to the client
- form is filled with the valid name

**How To Test**
- check that forms name fields are autofilled with a valid name on subscription change

![image](https://user-images.githubusercontent.com/21695878/62565992-d3a70e00-b83c-11e9-92c6-46a4fe031a85.png)
(name is project name + unix timestamp)
